aepp/flowservice.py

```diff
@@ -1356,6 +1406,24 @@ def getLandingZoneStorageAccountName(
         """
         return self.getLandingZoneCredential(dlz_type=dlz_type)["storageAccountName"]
 
+    def getLandingZoneStorageName(
```

Comment:
The name is not very explicit about what is being returned, I think `getLandingZoneStorageAccountName` made sense and works for both S3 and Blob-based DLZ. Can we keep it in the same function and not mark it deprecated instead of having another function?

Reply:
i repourposed this one, we'll have:
getLandingZoneStorageName - will return either containerName for azure and upper-most foler for aws